docs/topics/about-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+// * docs/idea-plugin-guide/master.adoc
+
+
+[id='about-intellij-idea-plugin_{context}']
+= About the {ProductShortName} plugin for IntelliJ IDEA
+
+The {ProductName} ({ProductShortName}) plugin for IntelliJ IDEA helps you migrate and modernize applications.
```

Comment:
It's a bit jarring to see "plugin" used in the first sentence and "extension" in the next sentence. I realize that different IDEs might use different terms (plugin vs extension). If that is the case, you may want to consider using a local attribute. In your assembly file (master.adoc), add an attribute like `:plugin: plugin` or `plugin: extension`. Then use `{plugin}` in the module. I've been doing that a lot lately in order to avoid excessive numbers of ifdefs. If only one word or phrase needs to be changed, a local attribute makes sense. I avoided them for a long time because I was not sure whether they were OK with OpenShift docs; local attributes are not documented in their manual or contributors' guide. But Vikram says it's OK.

Reply:
I should have used "plugin" throughout the section. Will fix.